pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -180,6 +186,12 @@ PipelineConfig createBatchOptions() {
     options.setViewDefinitionsDir(Strings.nullToEmpty(viewDefinitionsDir));
     options.setSinkDbConfigPath(Strings.nullToEmpty(sinkDbConfigPath));
 
+    if (!Strings.isNullOrEmpty(fhirSinkPath)) {
+      options.setFhirSinkPath(fhirSinkPath);
+      options.setSinkUserName(sinkUserName);
```

Comment:
nit: Please use `nullToEmpty()` on `sinkUserName` and `sinkPassword` when setting the options here.